spacy/tests/package/test_requirements.py

```diff
@@ -22,6 +22,7 @@ def test_build_dependencies():
     ]
     # ignore language-specific packages that shouldn't be installed by all
     libs_ignore_setup = [
+        "numpy",
```

Comment:
Why is `numpy` in `libs_ignore_requirements` and `libs_ignore_setup` now?

Reply:
The test can't deal with the duplicate entries.

Reply:
Probably also not the diverging versions in build vs. install requirements, which we didn't have before.